api/PeriodicSyncManager.json

```diff
@@ -0,0 +1,240 @@
+{
+  "api": {
+    "PeriodicSyncManager": {
+      "__compat": {
+        "support": {
+          "chrome": {
+            "version_added": "80"
+          },
+          "chrome_android": {
+            "version_added": "80"
+          },
+          "edge": {
+            "version_added": "80"
+          },
+          "firefox": {
+            "version_added": false
+          },
+          "firefox_android": {
+            "version_added": false
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "67"
+          },
+          "opera_android": {
+            "version_added": "57"
+          },
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": false
+          },
+          "samsunginternet_android": {
+            "version_added": false
+          },
+          "webview_android": {
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "PeriodicSyncManager": {
```

Comment:
The spec doesn't describe a constructor for this interface. Is this… real? Some more information about how you got the data would be especially helpful here.

Reply:
Testing through Chrome's console implies there's a constructor, but attempting to run it produces an "illegal constructor" error, so there isn't actually one.  Fixing.